From 73eaf517702a448a4b073ca666413f030ffa8fe1 Mon Sep 17 00:00:00 2001
From: Daniel van Vugt <daniel.van.vugt@canonical.com>
Date: Thu, 10 Oct 2019 17:21:13 +0800
Subject: [PATCH 02/50] cogl-pango/meson.build: Remove extraneous quoting

It wasn't necessary (see other instances of -DG_LOG_DOMAIN) and somewhere
along the line it was getting turned into forward slashes becoming a syntax
error:

```
/usr/include/glib-2.0/gobject/gobject.h:767: syntax error, unexpected '/' in
...
g_assertion_message (/"CoglPango/",
```

https://gitlab.gnome.org/GNOME/mutter/merge_requests/841
---
 cogl/cogl-pango/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cogl/cogl-pango/meson.build b/cogl/cogl-pango/meson.build
index 787ec0132..06fba5195 100644
--- a/cogl/cogl-pango/meson.build
+++ b/cogl/cogl-pango/meson.build
@@ -62,7 +62,7 @@ if have_introspection
     ],
     extra_args: introspection_args + [
       '-UCOGL_COMPILATION',
-      '-DG_LOG_DOMAIN=\"CoglPango\"',
+      '-DG_LOG_DOMAIN="CoglPango"',
     ],
     install_dir_gir: pkglibdir,
     install_dir_typelib: pkglibdir,
-- 
2.23.0

